modules/hal_nordic/nrfx/Kconfig.logging

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menu "nrfx drivers logging"
+	depends on LOG
+
+config LOG_NRFX_ADC
```

Comment:
I think it would be more adequate to use `NRFX_*_LOG` naming scheme. These options enable logging for particular nrfx drivers, not somehow activate those drivers within the logging subsystem.